src/node_messaging.cc

```diff
@@ -472,21 +472,13 @@ MessagePort::MessagePort(Environment* env,
   Debug(this, "Created message port");
 }
 
-void MessagePort::AddToIncomingQueue(Message&& message) {
-  data_->AddToIncomingQueue(std::move(message));
-}
```

Comment:
how is this change related?

Reply:
It’s not, it’s just something that I noticed, and putting it into another PR would have lead to yet another merge conflict. I’ve split it into a separate commit, as @bnoordhuis suggested.

Reply:
thanks. part of my question (which I meant to ask but did not come out like that) was why this change required at all - that is not covered in the PR description. But I got it now as: it is an unused piece of code.